config/test.yml.erb

```diff
@@ -44,3 +44,9 @@ disable_s3_image_uploads: true
 
 # provide a unique path for firebase channels data for ci, to avoid conflicts in channel ids.
 firebase_channel_id_suffix: '<%=ci ? "-#{circle_run_identifier}" : ''%>'
+
+# Configurations for making calls to the Immersive Reader API on Azure
+imm_reader_tenant_id:     '01521c57-6696-4ac9-b8ba-d2f2955893bd'
+imm_reader_client_id:     '92ef2120-f523-41fe-a995-f9720bf70b81'
+imm_reader_client_secret: !Secret
```

Comment:
The `test` environment generally should not require secrets to function. (This currently makes the CI tests fail because that environment is not provided AWS credentials with permission to access secrets.)

Reply:
Oh thanks! You saved me a bunch of time trying to debug that.